packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -130,6 +143,22 @@ const VideoEffectsPaneTrampoline = (
   selectableVideoEffects?: _VideoEffectsItemProps[],
   onEffectChange?: (effectKey: string) => Promise<void>
 ): JSX.Element => {
+  /* @conditional-compile-remove(video-background-effects) */
+  const [dismissedError, setDismissedError] = useState<DismissedError>();
+  /* @conditional-compile-remove(video-background-effects) */
+  const latestEffectError = useSelector(videoBackgroundErrorsSelector);
+  /* @conditional-compile-remove(video-background-effects) */
+  let showError = false;
```

Comment:
nitty nit: consider ternary to avoid mutable `let` usage (we opt for const where possible as it lets the reader of the code know this is the only place it gets set and won't have changing values later on)

```ts
const showError = latestEffectError && (latestEffectError.timestamp > dismissedError.dismissedAt);
```